src/libsvg/util.cc

```diff
@@ -1,27 +1,95 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016-2018, Torsten Paul <torsten.paul@gmx.de>,
+ *                          Marius Kintel <marius@kintel.net>
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the "Software"),
+ * to deal in the Software without restriction, including without limitation
+ * the rights to use, copy, modify, merge, publish, distribute, sublicense,
+ * and/or sell copies of the Software, and to permit persons to whom the
+ * Software is furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
 #include <boost/spirit/include/qi.hpp>
 
 #include "util.h"
 
+BOOST_FUSION_ADAPT_STRUCT(libsvg::length_struct, (double, number) (std::string, unit))
```

Comment:
according to travis, ubuntu trusty (14.04) seems to not like this:
```
src/libsvg/util.cc:29:50: error: expected identifier before ‘(’ token
 BOOST_FUSION_ADAPT_STRUCT(libsvg::length_struct, (double, number) (std::string, unit))
                                                  ^
src/libsvg/util.cc:29:59: error: ‘number’ has not been declared
 BOOST_FUSION_ADAPT_STRUCT(libsvg::length_struct, (double, number) (std::string, unit))
                                                           ^
```
Ubuntu 14.04 should use boost 1.54. boost 1.54 should have BOOST_FUSION_ADAPT_STRUCT
https://www.boost.org/doc/libs/1_54_0/libs/fusion/doc/html/fusion/adapted/adapt_struct.html

Maybe a missing include?
```
#include <boost/fusion/adapted/struct/adapt_struct.hpp>
#include <boost/fusion/include/adapt_struct.hpp>
```
The boost headers seams to include a lot of boost headers them self, but this change change with different versions.

Reply:
Yep, lets see if that helps.